### ISSUE TITLE
"export 'createElement' (imported as 'h') was not found in '@vue/composition-api'

### ISSUE THREAD

**Issue by 542154968** [luwanquan/babel-preset-vca-jsx#15]
"export 'createElement' (imported as 'h') was not found in '@vue/composition-api'

![image](https://user-images.githubusercontent.com/25705659/87213788-12e9b300-c35a-11ea-9baa-e5cf429d5111.png)

```json
{
  "name": "composition-api-vue2",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "serve": "vue-cli-service serve",
    "build": "vue-cli-service build",
    "lint": "vue-cli-service lint"
  },
  "dependencies": {
    "@vue/composition-api": "^1.0.0-beta.3",
    "core-js": "^3.6.5",
    "vue": "^2.6.11",
    "vue-router": "^3.2.0",
    "vuex": "^3.4.0"
  },
  "devDependencies": {
    "@vue/cli-plugin-babel": "~4.4.0",
    "@vue/cli-plugin-eslint": "~4.4.0",
    "@vue/cli-plugin-router": "~4.4.0",
    "@vue/cli-plugin-vuex": "~4.4.0",
    "@vue/cli-service": "~4.4.0",
    "@vue/eslint-config-prettier": "^6.0.0",
    "babel-eslint": "^10.1.0",
    "babel-preset-vca-jsx": "^0.3.5",
    "eslint": "^6.7.2",
    "eslint-plugin-prettier": "^3.1.3",
    "eslint-plugin-vue": "^6.2.2",
    "node-sass": "^4.12.0",
    "prettier": "^1.19.1",
    "sass-loader": "^8.0.2",
    "vue-template-compiler": "^2.6.11"
  }
}

```

**Comment by zhongqf** [comment]
#12

**Comment by 542154968** [comment]
> #12

不行
![image](https://user-images.githubusercontent.com/25705659/87215456-349e6680-c369-11ea-8982-8b3d5621492d.png)

**Comment by joaomede** [comment]
this fix:
```json
"babel-preset-vca-jsx": "b12f/babel-preset-vca-jsx#patch-1",
```

**Comment by luwanquan** [comment]
Fixed, see v0.3.6
